app/registries/foreman/access_permissions.rb

```diff
@@ -721,6 +721,13 @@
       :"api/v2/personal_access_tokens" => [:destroy]
   end
 
+  permission_set.security_block :json_web_tokens do |map|
+    map.permission :generate_json_web_tokens,
+      :"api/v2/json_web_tokens" => [:generate]
+    map.permission :invalidate_json_web_tokens,
+      :"api/v2/json_web_tokens" => [:invalidate]
```

Comment:
**Just thinking outloud, feel free to ignore if you have a reson for this**
do we need two perms for it? shouldn't I be able to invalidate if I can generate and vice versa?